src/template-tags/tickets.php

```diff
@@ -194,24 +194,38 @@ function tribe_events_ticket_is_on_sale( Tribe__Tickets__Ticket_Object $ticket )
 	/**
 	 * Gets the "tickets sold" message for a given ticket
 	 *
-	 * @param Tribe__Ticket_Object $ticket Ticket to analyze
+	 * @param Tribe__Tickets__Ticket_Object $ticket Ticket to analyze
 	 *
 	 * @return string
 	 */
-	function tribe_tickets_get_ticket_stock_message( $ticket ) {
-		$stock = $ticket->original_stock();
-		$sold = $ticket->qty_sold();
-		$pending = $ticket->qty_pending();
-
-		$pending_message = '';
-		if ( $pending > 0 ) {
-			$pending_message = sprintf( _n( '(%d awaiting review)', '(%d awaiting review)', $pending, 'event-tickets' ), (int) $pending );
+	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket ) {
+		$stock     = $ticket->stock();
+		$sold      = $ticket->qty_sold();
+		$cancelled = $ticket->qty_cancelled();
+		$pending   = $ticket->qty_pending();
+
+		// There may not be a fixed inventory - in which case just report the number actually sold so far
+		if ( empty( $stock ) && $stock !== 0 ) {
+			$message = sprintf( esc_html__( 'Sold %d', 'event-tickets' ), esc_html( $sold ) );
 		}
-
-		if ( ! $stock ) {
-			$message = sprintf( esc_html__( 'Sold all %1$d %2$s', 'event-tickets' ), esc_html( $sold ), esc_html( $pending_message ) );
-		} else {
-			$message = sprintf( esc_html__( 'Sold %1$d of %2$d %3$s', 'event-tickets' ), esc_html( $sold ), esc_html( $stock ), esc_html( $pending_message ) );
+		// If we do have a fixed stock then we can provide more information
+		else {
+			$cancelled_count = empty( $cancelled ) ? '' : esc_html( sprintf(
+				_x( ' %1$s %2$d', 'ticket stock message (cancelled stock)', 'event-tickets' ),
+				_x( 'cancelled:', 'ticket stock message (cancelled stock)', 'event-tickets' ),
+				(int) $cancelled
```

Comment:
I'm guessing I'm not seeing something obvious, but out of curiosity why do this:

```php
sprintf(
  _x( ' %1$s %2$d', 'ticket stock message (cancelled stock)', 'event-tickets' ),
  _x( 'cancelled:', 'ticket stock message (cancelled stock)', 'event-tickets' ),
  (int) $cancelled
)
```

Rather than this:

```php
sprintf(
  _x( ' cancelled: %1$d', 'ticket stock message (cancelled stock)', 'event-tickets' ),
  (int) $cancelled
)
```